packages/paste-core/utilities/grid/src/Grid.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {useTheme} from '@twilio-paste/theme';
+import {ThemeShape} from '@twilio-paste/theme-tokens';
+import {MarginProps, Space, SpaceProps} from '@twilio-paste/style-props';
+import {safelySpreadBoxProps} from '@twilio-paste/box';
+import {Flex, Vertical} from '@twilio-paste/flex';
+
+export interface GridProps extends SpaceProps {
+  children: NonNullable<React.ReactNode>;
+  gutter?: Space;
+  vertical?: Vertical;
+}
+
+const getNegativeMargin = ({gutter}: GridProps, theme: ThemeShape): {} => {
+  if (Array.isArray(gutter)) {
+    return (gutter as []).map((value: string) => {
+      return `-${theme.space[value]}`;
+    });
+  }
+
+  if (gutter) {
+    return `-${theme.space[gutter as string]}`;
+  }
+
+  return 'auto';
+};
+
+const getGridStyles = (props: GridProps): MarginProps => {
+  const theme = useTheme();
```

Comment:
For clarity, let's make theme a required argument to this function and put this line inside the function on line 50